### PR TITLE
Transfer learning

### PR DESCRIPTION
**Pull request by kordc**

### CODE REVIEW

**Comment by kordc** [comment]
https://github.com/SebChw/art/pull/150

**Comment by SebChw** [comment]
I get device mismatch error from the beginning of using HF model
image

**Review by SebChw — CHANGES_REQUESTED** [review]

File: {{cookiecutter.project_slug}}/tutorial.ipynb [hunk 1]
```diff
@@ -0,0 +1,814 @@
...
+    "We'll do everything in a script, your task will be to fill the `run.py` accordingly with our instructions from this tutorial.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install art nltk wordcloud"
```
[the remark this answers is not in the document]
Comment (kordc) [reply {{cookiecutter.project_slug}}/tutorial.ipynb:22]: Oh sure... I'll do `art-training` install

File: {{cookiecutter.project_slug}}/tutorial.ipynb [hunk 2]
```diff
@@ -0,0 +1,814 @@
...
+    "    early_stopping = EarlyStopping('CrossEntropyLoss-validate', patience=6)\n",
+    "    project.add_step(TransferLearning(model,\n",
+    "                                      freezed_trainer_kwargs={\"max_epochs\": 3,\n",
+    "                                                              \"check_val_every_n_epoch\": 2,\n",
+    "                                                              \"callbacks\": [early_stopping]},\n",
+    "                                      unfreezed_trainer_kwargs={\"max_epochs\": 3,\n",
+    "                                                                \"check_val_every_n_epoch\": 2,\n",
+    "                                                                \"callbacks\": [early_stopping]},\n",
+    "                                      freeze_names=[\"bert\"],\n",
+    "                                      logger=logger\n",
```
[the remark this answers is not in the document]
Comment (kordc) [reply {{cookiecutter.project_slug}}/tutorial.ipynb:738]: Yeah, I deleted the logger but not the argument itself, thanks

File: {{cookiecutter.project_slug}}/models/bert.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        # Define the tokenizer
+        self.tokenizer = AutoTokenizer.from_pretrained("prajjwal1/bert-tiny")
+
+    def parse_data(self, data):
+        # Parse the data so that it fits the model
+
+        # Get the batch
+        batch = data[BATCH]
+        # Tokenize the text
+        inputs = self.tokenizer(
```
[the remark this answers is not in the document]
Comment (kordc) [reply {{cookiecutter.project_slug}}/models/bert.py:36]: Okay, I'll try

**Pull request closed without merging**